docs/griptape-tools/official-tools/griptape-cloud-knowledge-base-client.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+The `GriptapeCloudKnowledgeBaseClient` is an advanced Tool designed to streamline how organizations access and search their data. It leverages a RAG pipeline and vector stores hosted in [Griptape Cloud](https://cloud.griptape.ai) and allows users to perform intuitive searches across a centralized [Knowledge Base](https://cloud.griptape.ai/knowledge-bases) that consolidates various data sources such as Confluence, Google Docs, and web pages. This integration empowers users to connect and search through information pools, making it easier to find relevant data quickly.
+
+Griptape utilizes this technology to enhance organizational efficiency and knowledge sharing. By connecting diverse documentation and data sources into a single searchable repository, the `GriptapeCloudKnowledgeBaseClient` helps facilitate faster decision-making while improving productivity.
```

Comment:
This feels a little marketing-ey for the docs. Can we be a bit more direct and say it connects to the RAG service on GT Cloud?

Reply:
Sure, will simplify.